test/Concurrency/DeferredSendableChecking/region_based_sendability.swift

```diff
@@ -327,3 +327,124 @@ actor A_Sendable {
         await a.foo(captures_self)
     }
 }
+
+@available(SwiftStdlib 5.1, *)
```

Comment:
to avoid needing all these availability annotations you can add -disable-availability-checking  to the swift-frontend invocation